README.md

```diff
@@ -16,6 +16,13 @@ Second, read the [Troubleshooting Checklist](https://github.com/Homebrew/brew/bl
 
 **If you don't read these it will take us far longer to help you with your problem.**
 
+## Contributing
+We'd love you to contribute to Homebrew. First, Please read our [Contribution guide](https://github.com/Homebrew/brew/blob/master/.github/CONTRIBUTING.md) and [Code of Conduct](https://github.com/Homebrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct).
+
+We explicitly welcome contributions from people who have never contributed to open-source before: we were all beginners once! We can help take a partially working pull request and help you get it merged. We are also actively seeking to diversify our contributors and especially welcome contributions from women from all backgrounds and people of colour.
```

Comment:
Yes, it's certainly a challenge to make this sound tangible and not overly clunky without unintentionally excluding anyone, though perhaps women and people of color are the majority among the minority/minorities we're addressing here. I also really like the phrase “people from all walks of life” that I've increasingly heard being used in similar contexts.